api/swagger.yaml

```diff
@@ -0,0 +1,119 @@
+openapi: 3.0.2
+info:
+  title: Web Server
+  description: A simple web server for managing users
+  contact:
+    email: dmanor@redhat.com
+  version: "1.0"
+servers:
+- url: http://127.0.0.1:3000
+paths:
+  /api/users:
+    get:
+      summary: Retrieves all the users from the system
+      operationId: findUsers
+      responses:
+        "200":
+          description: Users retrieved successfully
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/User'
+                x-content-type: application/json
+        "405":
```

Comment:
wait. I don't think 405 means that
in general, 4xx codes are an indication of the client doing something wrong, in oppose to the service having faulty condition